.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'extends': 'airbnb',
+  'parser': 'babel-eslint',
+  'env': {
+    'jest': true,
+  },
+  'rules': {
+    'no-use-before-define': 'off',
+    'react/jsx-filename-extension': 'off',
+    'react/prop-types': 'off',
```

Comment:
Lets require prop-types. Makes it easier for others to know what props are required for a component.